README_new.md

```diff
@@ -0,0 +1,132 @@
+<p align="center">
+<img src="logo.png">
+</p>
+
+
+babylon health
+==================================
+
+At babylon, we firmly believe that **transparency** and **flexibility** are core values that should be present in everything we do, and hiring the best person should reflect that belief.
+
+That's why we in the iOS team have open-sourced our recruitment process, for a fairer assessment process and so that candidates know exactly what to expect, all the way to the [questions we may ask during a face-to-face interview](TODO).
+
+We also aim to be as flexible and inclusive as possible, equally accepting applications from candidates who want to work on-site at our office in London, or who want to work remotely, or who have a limited amount of time to write a demo project, or who have open-source contributions.
+
+### Our expectations
+
+babylon aims to hire only the most qualified people available at any given time. That doesn't mean that a candidate must have several academic degrees or have years and years of experience. We value **potential** equally as much as **knowledge** and **experience**. Above all, we value one's ability to **think independently** and to **engage**. As effective as they may be, we don't want robots.
+
+We also want to hire people who are **reasonable**, **professional**, and **fun** to work with. After all, they are people we will spend a great deal of time interacting with on a daily basis.
+
+As such, these are the main areas the iOS team will focus our assessment of candidates on:
+
+- Technical expertise
+ - Languages (Swift, Objective-C) and platforms (iOS, watchOS, tvOS, macOS)
+ - Architectural design patterns (MVC, MVVM, Viper, etc)
+ - Knowledge of iOS APIs
+ - Knowledge of the development cycle
+ - Knowledge of reactive programming
+ - Interview demo project
+- Interest in sharing one's knowledge
+- Interest in learning new knowledge
+- Interest in keeping oneself up to date
+- Ability to think and work independently
+- Ability to work in a large team
+- Professional and personal maturity
+- Overall attitude
+- Likelihood of fitting in well with the company's culture
+- Is the candidate someone we would like to work with on a daily basis?
+
+### The iOS recruitment process
+
+Our recruitment process is different between candidates who are interested in working remotely and candidates who are interested in working on-site at our office in London, in **two** ways:
+- remote candidates are *required* to be proficient in **reactive programming** (`Reactive Swift`/`Reactive Cocoa` or `RxSwift`) at an above-average level
```

Comment:
[Nit] _ReactiveSwift_ and _ReactiveCocoa_.

Reply:
👍